deploy/bundle/manifests/volumereplication.crd.yaml

```diff
@@ -0,0 +1,165 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
is this a duplicate copy of the CRD? if there are any changes in the CRD in the volume replication operator. Do the same changes need to be done here also?

Reply:
No. Changes go to Rook. We pull it from Rook image.

Reply:
Rook pulls the CRDs from the volume-replication github repo (and so do we DS) so there is no need to update them in the rook repo.